src/Services/Catalog/Catalog.API/Controllers/CatalogController.cs

```diff
@@ -207,39 +154,38 @@ public async Task<IActionResult> CatalogTypes()
         [ProducesResponseType(typeof(List<CatalogItem>), (int)HttpStatusCode.OK)]
         public async Task<IActionResult> CatalogBrands()
         {
-            var items = await _catalogContext.CatalogBrands
-                .ToListAsync();
-
-            return Ok(items);
+            var totalCatalogBrands = await _catalogService.ListAllCatalogBrandAsync();
+            return Ok(totalCatalogBrands);
         }
 
         //PUT api/v1/[controller]/items
         [Route("items")]
         [HttpPut]
         [ProducesResponseType((int)HttpStatusCode.NotFound)]
         [ProducesResponseType((int)HttpStatusCode.Created)]
-        public async Task<IActionResult> UpdateProduct([FromBody]CatalogItem productToUpdate)
+        public async Task<IActionResult> UpdateProduct([FromBody]CatalogItem itemToUpdate)
         {
-            var catalogItem = await _catalogContext.CatalogItems
-                .SingleOrDefaultAsync(i => i.Id == productToUpdate.Id);
+            List<CatalogItem> totalCatalogItems = (await _catalogService.ListAllAsync()).ToList();
+
+            var catalogItem = totalCatalogItems.SingleOrDefault(i => i.Id == itemToUpdate.Id);
 
             if (catalogItem == null)
             {
-                return NotFound(new { Message = $"Item with id {productToUpdate.Id} not found." });
+                return NotFound(new { Message = $"Item with id {itemToUpdate.Id} not found." });
             }
 
             var oldPrice = catalogItem.Price;
-            var raiseProductPriceChangedEvent = oldPrice != productToUpdate.Price;
+            var raiseProductPriceChangedEvent = oldPrice != itemToUpdate.Price;
```

Comment:
Change Variable name to  PriceChanged

Reply:
oldPrice will be renamed as PriceChanged

Reply:
it was actually the raiseProductPriceChangedEvent  that we wanted to change.
oldPrice was adequate.